config/routes.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 Spree::Core::Engine.routes.draw do
-  # Add your extension routes here
+  post '/api/order/pay_tomorrow', to: '/solidus_pay_tomorrow/create_order#create', as: 'create_pay_tomorrow_order'
```

Comment:
I think the API namespace here is a typo

Reply:
This is not clear, because the Controller inherits from Spree::API I'm resolving this conversation and I'll try to get more details on the Requested change message.